examples/step-50/doc/results.dox

```diff
@@ -1,98 +1,261 @@
+  <td align="right">17</th>
+  <td align="right">65M</th>
+  <td align="right">0.22</th>
+  <td></th>
+  <td align="right">1.122</th>
+  <td align="right">0.686</th>
+  <td align="right">0.309</th>
+  <td></th>
+  <td align="right">2.117</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">7168</th>
+  <td align="right">19</th>
+  <td align="right">256M</th>
+  <td align="right">0.16</th>
+  <td></th>
+  <td align="right">1.214</th>
+  <td align="right">0.893</th>
+  <td align="right">0.521</th>
+  <td></th>
+  <td align="right">2.628</th>
+</tr>
+<tr>
+  <th align="right">MB</th>
+  <td align="right">112</th>
+  <td align="right">13</th>
+  <td align="right">4M</th>
+  <td align="right">0.37</th>
+  <td></th>
+  <td align="right">1.714</th>
+  <td align="right">2.934</th>
+  <td align="right">0.716</th>
+  <td></th>
+  <td align="right">5.364</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">448</th>
+  <td align="right">15</th>
+  <td align="right">16M</th>
+  <td align="right">0.29</th>
+  <td></th>
+  <td align="right">1.927</th>
+  <td align="right">3.776</th>
+  <td align="right">1.190</th>
+  <td></th>
+  <td align="right">6.893</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">1792</th>
+  <td align="right">17</th>
+  <td align="right">65M</th>
+  <td align="right">0.22</th>
+  <td></th>
+  <td align="right">2.171</th>
+  <td align="right">4.862</th>
+  <td align="right">1.660</th>
+  <td></th>
+  <td align="right">8.693</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">7168</th>
+  <td align="right">19</th>
+  <td align="right">256M</th>
+  <td align="right">0.16</th>
+  <td></th>
+  <td align="right">2.386</th>
+  <td align="right">7.260</th>
+  <td align="right">2.560</th>
+  <td></th>
+  <td align="right">12.206</th>
+</tr>
+<tr>
+  <th align="right">AMG</th>
+  <td align="right">112</th>
+  <td align="right">13</th>
+  <td align="right">4M</th>
+  <td align="center">-</th>
+  <td></th>
+  <td align="right">1.460</th>
+  <td align="right">0.405</th>
+  <td align="right">0.924</th>
+  <td></th>
+  <td align="right">2.789</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">448</th>
+  <td align="right">15</th>
+  <td align="right">16M</th>
+  <td align="center">-</th>
+  <td></th>
+  <td align="right">1.544</th>
+  <td align="right">0.456</th>
+  <td align="right">1.150</th>
+  <td></th>
+  <td align="right">3.150</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">1792</th>
+  <td align="right">17</th>
+  <td align="right">65M</th>
+  <td align="center">-</th>
+  <td></th>
+  <td align="right">1.654</th>
+  <td align="right">0.546</th>
+  <td align="right">1.460</th>
+  <td></th>
+  <td align="right">3.660</th>
+</tr>
+<tr>
+  <th></th>
+  <td align="right">7168</th>
+  <td align="right">19</th>
+  <td align="right">256M</th>
+  <td align="center">-</th>
+  <td></th>
+  <td align="right">1.844</th>
+  <td align="right">1.010</th>
+  <td align="right">1.890</th>
+  <td></th>
+  <td align="right">4.744</th>
```

Comment:
AMG does not seem that bad?

Reply:
AMG is quite good, yes. Partly because there is no inherent imbalance and partly because this problem is very AMG-friendly (Q2 Laplace).